samples/net/mqtt/doc/description.rst

```diff
@@ -173,6 +173,25 @@ The following serial UART output is displayed in the terminal emulator:
       [00:04:01.475,982] <inf> transport: Publishing message: "Hello MQTT! Current uptime is: 241458" on topic: "my/publish/topic"
       [00:05:01.475,982] <inf> transport: Publishing message: "Hello MQTT! Current uptime is: 301459" on topic: "my/publish/topic"
 
+Sample output using LTE with IPv6:
```

Comment:
```suggestion
The following sample output is displayed using LTE with IPv6:
```
Do we need to follow the same testing steps mentioned in Testing section to get this sample output (using LTE with IPv6)?

Reply:
Yes, same testing steps, just with a different build configuration/device. I added the details of the specific build command (i.e. for the Thingy:91) that matches the sample output.

Different configurations will have slightly different output, but they all now support IPv6, i.e. IPv6 will now also work with Wi-Fi. Actually, now that I have made the IP address logging at DBG level, the output will not look any different than it did before, unless you turn on that additional level.